sorc/checkout.sh

```diff
@@ -99,4 +99,13 @@ else
     echo 'Skip. Directory verif-global.fd already exist.'
 fi
 
+echo wcoss2_ecf_cycled checkout ...
+if [[ ! -d wcoss2_ecf_cycled ]] ; then
+    rm -f ${topdir}/checkout-wcoss2_ecf_cycled.log
+    git clone --branch feature/wcoss2_ecf_cycled https://github.com/lgannoaa/ecflow.git wcoss2_ecf_cycled >> ${topdir}/checkout-wcoss2_ecf_cycled.log 2>&1
+    cd ${topdir}
+else
+    echo 'Skip. Directory wcoss2_ecf_cycled already exist.'
+fi
```

Comment:
What is this and why are these scripts to run ecflow not in the global-workflow repository?

More importantly, this is a private repository!!!

![Screen Shot 2022-03-23 at 5 39 08 PM](https://user-images.githubusercontent.com/11394126/159800276-e94ab8e5-ff95-4f67-82a9-830bf02046ed.png)

Reply:
Checking out package and link is standard practice in global-workflow. We should do the same for ecf directory. Maybe we can create an official repo for ecflow?

Reply:
Storing files in a Private repository is not standard practice anywhere in NOAA-EMC.
Has a need been shown for having a separate set of scripts;  let alone in a users personal account? 
Adding a repository with ecf files is outside of the scope of this PR that enables cycled parallel on WCOSS2 with the verification components.

Reply:
Never said storing file in private repository is a standard practice. Hence the suggestion to create a official repo for ecflow workflow files.
As requested, I will modify this approach to overwrite ecf directory under $HOMEgfs to the new version.

Reply:
Yes. Please open a separate PR with those changes related internal to `ecf/`
I would like to see the changes in context of NCO's use of them.

Reply:
I want to minimize the number of repos that are used to check out a full application from, this is making the global workflow unsustainable. A repo should only really be created when it can stand on its own and be run. Otherwise it should be in one repository only

Reply:
@aerorahul This PR will be the PR for ecflow. I do not understand why it need to have another new PR.

Reply:
@arunchawla-NOAA Thank you for guidance.